modules/samples/default/decision-tree/README.MD

```diff
@@ -7,7 +7,9 @@ Algorithm: Decision-Tree
 Before Running
 ==============
 
-* Start WSO2 Machine Learner product with a fresh database.
+* Start WSO2 Machine Learner product.
+
+* Make sure jq is installed.
```

Comment:
jq command line tool has been used in the model-generation.sh script to read values from the json responses. (ie. to get the projectId, analysisId, etc.)